zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/tasks/archiver/OpenSearchArchiverRepository.java

```diff
@@ -247,7 +247,7 @@ private ArchiveBatch createArchiveBatch(final SearchResponse<?> response, final
             .map(Hit::id)
             .toList();
 
-    metrics.recordArchivingBatchSize(ids.size());
+    metrics.recordProcessInstancesArchiving(ids.size());
```

Comment:
:question: is this correct? is different than the Elasticsearch repository and I noticed that `recordProcessInstancesArchiving` is called from `ProcessInstancesArchiverJob`

Reply:
Good catch. This was an oversight. We can remove it here as it is part of the job.